### PR TITLE
Change zoom on image preview bis

### PR DESCRIPTION
**Pull request by AlexandreSi**
- Resolves https://github.com/4ian/GDevelop/issues/4890
- Changes technical strategy for points preview to use the same as for collision masks preview
- Improves mobile user experience:
  - One can move touching the screen
  - One can zoom pinching the screen

TODO:
- ✅ Make sure it works with the physics2Editor preview

Tested on:
- macOs:
  - Chrome
  - Safari
  - Firefox
- iPad:
  - Chrome
  - Safari
- Android:
  - Chrome 

There's an issue on safari that makes it impossible to select points outside of the svg element (so outside of the image).
I think that using the points editor on the side is a decent workaround.

Stories: https://gdevelop-storybook.s3.amazonaws.com/change-zoom-on-image-preview-bis/latest/index.html?path=/story/objecteditor-spriteeditor--points

### CODE REVIEW

**Comment by ClementPasteau** [comment]
Just by playing with the story, i'm wondering if the scrolling speed should be slightly faster.
When zoomed in, I spend ages trying to go from one side to the other of the image. (maybe *1.5 ?)

File: newIDE/app/src/ResourcesList/ResourcePreview/ImagePreview.js [hunk 1]
```diff
@@ -275,7 +412,12 @@ const ImagePreview = ({
             {!hideControls && (
               <MiniToolbar noPadding>
                 <IconButton
-                  onClick={() => zoomBy(zoomOutFactor)}
+                  onClick={() =>
+                    zoomAroundPointBy(zoomOutFactor, [
+                      (containerWidth || 0) / 2,
+                      (containerHeight || 0) / 2,
```
Comment (ClementPasteau) [inline newIDE/app/src/ResourcesList/ResourcePreview/ImagePreview.js:418]: took me a while to understand that you're zooming around the current frame center (and not the whole image center)
But it's great :)

**Review by ClementPasteau — COMMENTED** [review]
I'm afraid this will be quite tricky to jump into, next time we'll want to modify this without context.
But not sure we can do a lot better for the moment.
Great job 👍 👍

**Review by ClementPasteau — APPROVED** [review]

File: newIDE/app/src/ResourcesList/ResourcePreview/ImagePreview.js [hunk 2]
```diff
@@ -335,6 +335,11 @@ const ImagePreview = ({
     [imageResourceSource]
   );
 
+  const containerCenter = React.useMemo(
```
Comment (ClementPasteau) [inline newIDE/app/src/ResourcesList/ResourcePreview/ImagePreview.js:338]: ❤️

**Review by 4ian — APPROVED** [review]

**Pull request merged**